src/main/resources/templates/payload/PROPERTY_LISTED-event.ftl

```diff
@@ -0,0 +1,40 @@
+{
+"event": {
+  "type": "CCS_ADDRESS_LISTED",
+  "source": "FIELDWORK_GATEWAY",
+  "channel": "FIELD",
+  "dateTime": "${eventDate}",
+  "transactionId": "${ccsPropertyListingOutcome.transactionId}"
+},
+"payload": {
+  "CCSProperty": {
+    "collectionCase": {
+      "id": "${caseId}"
+    },
+    "interviewRequired": "${interviewRequired}",
```

Comment:
I dont know whether the boolean is interpreted as a string. It might be ok. i.e: "true" or true